script.js

```diff
@@ -58,62 +59,87 @@
     return uniq([...pluralized, ...possibleAliases])
   }
 
-  const enrichResources = (resources) =>
+   enrichResources = (resources) => (
```

Comment:
```suggestion
   const enrichResources = (resources) => (
```

While it's technically legal to define without a declaration, it's not a great practice.

Reply:
absolutely agree